.github/workflows/update-package-version.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -eu
+
+package_json_path=$1
+commit_hash=$2
+commit_short_hash=$( git rev-parse --short $commit_hash )
+
+version=$( cat $package_json_path | jq -r .version )
+version=$version-$commit_short_hash
```

Comment:
Please use it this way:
version=${version}-${commit_short_hash}

Might catch some issues if we change the format of the version.